src/Storageless/Http/SessionMiddleware.php

```diff
@@ -58,12 +58,14 @@ final class SessionMiddleware implements MiddlewareInterface
     private Configuration $config;
     private SetCookie $defaultCookie;
 
+    /** @param non-empty-string $sessionAttribute */
```

Comment:
Let's perhaps use `&literal-string` to avoid misuse

Reply:
You put a `&`, so I guess you want me to use both `non-empty-string` and `literal-string`:
```php
/** @param non-empty-string&literal-string $sessionAttribute */
```
but:
```
src/Storageless/Http/SessionMiddleware.php:61:16 - Intersection types must be all objects, Psalm\Type\Atomic\TNonEmptyString provided in docblock
```
BTW I don't see misuse possibilities here :thinking:

Reply:
argh, thought psalm supported this @_@ 

Perhaps `literal-string`  suffices then?

As for misuse: the point is to not randomly override other attributes, since attributes are usually quite fragile.